pages/reference.mdx

```diff
@@ -406,6 +408,7 @@ In the case of endpoints that support animated images, the hash will begin with
 | Soundboard Sounds                | /soundboard-sounds/sound_id                                                                                                                                                                        | MP3, OGG                  |
 | Video Filter                     | /users/[user_id](/resources/user#user-object)/video-filter-assets/video_filter_id/video_filter_asset.png                                                                                           | PNG, JPEG, GIF, MP4, WEBP |
 | Profile Badge                    | /badge-icons/[badge_icon](/resources/user#profile-badge-structure).png                                                                                                                             | PNG                       |
+| Quest Asset                      | /quests/[quest_id](/resources/quests#quest-object)/[theme](/resources/user/#client-theme).png                                                                                                      | PNG, JPEG, GIF, SVG, WebP |
```

Comment:
Isn't this also missing asset_name in the path (should link back to the asset structure)